src/main/resources/testchipip/csrc/mm_dramsim2.h

```diff
@@ -39,10 +39,11 @@ class mm_dramsim2_t : public mm_t
   mm_dramsim2_t(int axi4_ids) : 
       read_id_busy(axi4_ids, false),
       write_id_busy(axi4_ids, false) {};
-  mm_dramsim2_t(std::string ini_dir, int axi4_ids) :
+  mm_dramsim2_t(std::string ini_dir, int axi4_ids, uint64_t clock_hz = 0) :
```

Comment:
Is this a reasonable default?

Reply:
I guess it is due to this comment in dramsim:
`If cpuClkFreqHz == 0, then assume a 1:1 ratio`

Reply:
Yeah this results in the previous behavior.

Reply:
Under 100 MHz what ratio is simulated by default?

Reply:
Its 666 MHz : 100 MHz